scipy/integrate/_quadrature.py

```diff
@@ -991,38 +1240,38 @@ def romberg(function, a, b, args=(), tol=1.48e-8, rtol=1.48e-8, show=False,
 #    where k = N // 2
 #
 _builtincoeffs = {
-    1: (1,2,[1,1],-1,12),
-    2: (1,3,[1,4,1],-1,90),
-    3: (3,8,[1,3,3,1],-3,80),
```

Comment:
I would revert these cosmetic code changes because they are unrelated. I guess this was you IDE.

Reply:
Yup, this was unintentional. Reverted the change.